pkg/apis/networking/validation/validation.go

```diff
@@ -561,6 +568,55 @@ func validateIngressTypedLocalObjectReference(params *api.TypedLocalObjectRefere
 	return allErrs
 }
 
+// validateIngressClassParametersReference ensures that Parameters fields are valid.
```

Comment:
Can you add a reference to the original validation function used for the ObjectRef before this change in this comment just so it's clear that this has to be identical to that + with additions for scope field?

Reply:
I added a comment and also used the same function for validating the common fields (thereby removing some code duplication as well).